osrf/vrx-docker
# restore replay functionality

This PR updates the `vrx-docker` server so we can use the `replay_trial.bash` script to replay a run against a particular task. It resolves issue #65 . 

### To test
 
* Run through the [VRX Testing Tutorial](https://github.com/osrf/vrx/wiki/vrx_2023-testing) using `TEAM=example_team`. 
* Make sure to rebuild the `vrx-docker` image.
* When you run
   ```
  ./replay_trial.bash $TEAM $TASK $TRIAL
  ```
  you should see playback of the run. (Note that the example image waits until the running state to start moving).

### Notes
* This PR depends on https://github.com/osrf/vrx/pull/694 . I have temporarily modified the Dockerfile to build against the `M1chaelM/gz_recording_args` branch of the main repo until that is merged.
* Since this is the last feature missing for the [VRX Testing Tutorial](https://github.com/osrf/vrx/wiki/vrx_2023-testing), it also resolves https://github.com/osrf/vrx/issues/654 and https://github.com/osrf/vrx/issues/660.
